services/basic_types.proto

```diff
@@ -1592,3 +1593,32 @@ message StakingInfo {
         int64 staked_node_id = 6;
     }
 }
+
+message VirtualAddress {
+    /**
+     * The 20-byte EVM address.
+     */
+    bytes address = 1;
+
+    /**
+     * The ethereum transaction nonce associated with this address.
+     */
+    int64 nonce = 2;
+
+    /**
+     * Flag if this address should now be set or is the default address on the account.
```

Comment:
`VirtualAddress` shouldn't be for updates and only returned on queries. So set terminology doesn't make sense.

```suggestion
     * Flag if this address is the default address on the account.
```